caffe2/onnx/helper.h

```diff
@@ -16,24 +16,77 @@
 
 #pragma once
 
+#include "onnx/onnx_pb.h"
+
 #include <set>
 #include <string>
 #include <unordered_set>
 
-namespace caffe2 { namespace onnx {
+namespace caffe2 {
+namespace onnx {
+
+using ::ONNX_NAMESPACE::AttributeProto;
+using ::ONNX_NAMESPACE::NodeProto;
+
 class DummyName {
```

Comment:
Without documentation, a class with such a name looks strange

Reply:
I'll add document. On the other hand, I'll change this class in another PR as current design is using static variables which is not good.